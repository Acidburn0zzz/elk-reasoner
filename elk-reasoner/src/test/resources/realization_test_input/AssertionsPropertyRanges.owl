Prefix(:=<http://example.org/>)
Ontology(<www.example.org#assertions_property_ranges>
ObjectPropertyAssertion(:R :a :b)
SubObjectPropertyOf(:R :S)
ObjectPropertyRange(:S :B)
)
