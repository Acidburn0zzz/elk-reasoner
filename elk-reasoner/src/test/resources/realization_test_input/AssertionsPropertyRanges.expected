Prefix(:=<http://example.org/>)
Ontology(<www.example.org#assertions_property_ranges#taxonomy>
ClassAssertion(:B :b)
)